### PR TITLE
Fix button that has inner html

### PR DESCRIPTION
**Pull request by FrancisMengx**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes
Button should not have any html tag as children. In IE the inner html, would not render.

### CODE REVIEW

**Comment by FrancisMengx** [comment]
@dzearing Friendly ping.

**Comment by KevinTCoughlin** [comment]
FWIW I ran into this problem with #3636 -- I'd check `<Nav/>` because I think that's where this change would have the most impact.

**Comment by FrancisMengx** [comment]
@joschect this is currently breaking people picker in IE. The issue has not been addressed. Can we merge this to fix the current issue in find people?

File: packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx [hunk 1]
```diff
@@ -30,15 +30,15 @@ export class SuggestionsHeaderFooterItem extends BaseComponent<ISuggestionsHeade
   public render(): JSX.Element {
     const { renderItem, onExecute, isSelected, id } = this.props;
     return onExecute ? (
-      <button
+      <div
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx]: I don't know the context of this, but having a div that has a click target will not be focusable. I also am curious if a role here is missing, as this likely wouldn't be read by screen readers.

File: common/changes/office-ui-fabric-react/fix-button-that-has-inner-html_2018-08-03-18-49.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Button should not have any html tag as children. In IE the inner html, would not render. ",
```
Comment (dzearing) [inline common/changes/office-ui-fabric-react/fix-button-that-has-inner-html_2018-08-03-18-49.json]: This comment here needs to be adjusted. You're modifying pickers, not button. And even with the Pickers: prefix, what button are you talking about?

**Pull request merged**